src/views/Accessibility/AccessibilityRequestDetailPage.tsx

```diff
@@ -456,7 +470,19 @@ const AccessibilityRequestDetailPage = () => {
                 : message}
             </Alert>
           )}
-
+          {returnedUserErrors && (
+            <ErrorAlert
+              testId="create-accessibility-note-errors"
+              classNames="margin-bottom-4 margin-top-4"
+              heading="There is a problem"
+            >
+              {returnedUserErrors.map((
+                err: CreateAccessibilityRequestNote_createAccessibilityRequestNote_userErrors /* eslint-disable-line camelcase */
+              ) => {
+                return <p key={err.message}>{err.message}</p>;
```

Comment:
I'm looking through the AC and there's no mention of any specific error messages. It only mentions the copy "Error saving note."

Reply:
I think that specifically refers to server errors.  for `userErrors`, since they are something as a the result of user behavior, I think it makes sense to give them more information.  I don't think the AC was thinking about `userErrors`.